packages/reactive_ble_mobile/android/src/main/AndroidManifest.xml

```diff
@@ -9,17 +9,6 @@
         android:name="android.permission.BLUETOOTH_ADMIN"
         android:maxSdkVersion="30" />
 
-    <!-- required for API 23 - 30 -->
```

Comment:
So this is removed to prevent conflicts in apps using the plugin, but why should `android.permission.BLUETOOTH_CONNECT` be kept then? As `android.permission.BLUETOOTH_SCAN` is also removed below?

Reply:
Because `BLUETOOTH_CONNECT` doesn't have any possible modifiers. `BLUETOOTH_SCAN` has the optional `neverForLocation` modifier. I guess maybe there's a small chance someone using this plugin would only want to scan and not connect maybe? But that would probably be a very niche use-case.

Reply:
Usage will then become a bit weird.. Because users of the plugin **have to** specify the `SCAN` in their manifest, but not per-se the `CONNECT` because that's already done by the plugin. 

IMO I think we should then also not do `CONNECT` and update the readme that they'll have to add both `SCAN` and `CONNECT`/`ADVERTISE` (if they use it) to their manifest. But lemme check what the other maintainers think: 
@Taym95 @remonh87 Your opinion on this?

Reply:
I agree with @PieterAelse let's be consistent